arangod/RocksDBEngine/RocksDBSettingsManager.cpp

```diff
@@ -538,7 +348,7 @@ void RocksDBSettingsManager::loadSettings() {
         if (slice.hasKey("lastSync")) {
           _lastSync =
               basics::VelocyPackHelper::stringUInt64(slice.get("lastSync"));
-          LOG_TOPIC(TRACE, Logger::ENGINES)
+          LOG_TOPIC(ERR, Logger::ENGINES)
```

Comment:
IMHO this should not have log level ERR
```suggestion
          LOG_TOPIC(TRACE, Logger::ENGINES)
```